moped-etl/arcgis/components_to_agol.py

```diff
@@ -54,16 +63,15 @@ def make_esri_feature(*, esri_geometry_key, geometry, attributes):
     return feature
 
 
-def main():
-    logger.info("Getting token...")
-    get_token()
-
-    logger.info("Downloading component data...")
-    data = make_hasura_request(query=COMPONENTS_QUERY)["component_arcgis_online_view"]
+def make_all_features(data):
```

Comment:
this is some of the original code in this ETL that was repeated between full and partial replace routes of this script so i made a function out of it.